queries/kotlin/highlights.scm

```diff
@@ -24,6 +24,10 @@
 (super_expression
   "@" @variable.builtin)
 
+; NOTE: simplified pattern to replace (temporarily) problematic query that causes delay (see below)
+((simple_identifier) @property
```

Comment:
variable.member, yes no maybe?

Reply:
Yes, needs to be `@variable.member`; `@property` is a regression from the capture cleanup.